scripts/rpm_owners/packages.json

```
@@ -4579,6 +4579,13 @@
         "package": "vim",
         "status": "packaged"
     },
+    "vmfs6-tools": {
+        "added_by": "XCP-ng",
+        "maintainer": "OS Platform & Release",
+        "note": "as-is, not supported, not in theproduc, only in xcp-ng-labs",
```

Assuming that it's a typo and not a word I don't know :)

```suggestion
        "note": "as-is, not supported, not in the product, only in xcp-ng-labs",
```